### ISSUE TITLE
‖ scales too tall, compared to ⟨ and ⟩

### ISSUE THREAD

**Issue by mdeff** [stipub/stixfonts#225]
When ‖ is scaled to accommodate a subscripted letter, it looks too tall, compared to the angle brackets ⟨ and ⟩.

The following document, compiled with `latexmk -pdflua`, produces the following output.

```tex
\documentclass{article}
\usepackage{unicode-math}
\setmathfont{STIX Two Math}
\newcommand*{\norm}[1]{{\ensuremath{\left‖{#1}\right‖}}}
\newcommand*{\inner}[1]{{\ensuremath{\left⟨{#1}\right⟩}}}
\begin{document}
    \[
        \inner{f}_2^2            \hspace{2em}
        \norm{f}_2^2             \hspace{2em}
        \inner{f^2}_2^2          \hspace{2em}
        \norm{f^2}_2^2           \hspace{2em}
        \inner{\frac{1}{f}}_2^2  \hspace{2em}
        \norm{\frac{1}{f}}_2^2
    \]
\end{document}
```
![stix2](https://user-images.githubusercontent.com/6806065/147515752-55f45771-ad96-4968-aaa7-5866a1f04641.png)

Compare with Cambria (i.e., `\setmathfont{Cambria Math}`):
![cambria](https://user-images.githubusercontent.com/6806065/147515763-21c60152-3eac-442d-aa8d-5fbd7c29112e.png)

And Latin Modern (i.e., `\setmathfont{Latin Modern Math}`):
![lm_unicode](https://user-images.githubusercontent.com/6806065/147515768-75020609-07c3-4d60-9809-c873b9b1caed.png)

**Comment by tiroj** [comment]
Please confirm which version of the STIX Two Math font you are testing here. We did work coordinating the variations and heights of delimiters recently, so I will track this issue to see if we have succeeded in resolving the problem.

**Comment by mdeff** [comment]
Thanks for the quick feedback! I'm unsure how to check the version, but `fc-query STIXTwoMath-Regular.otf` reports `fontversion: 138281(i)(s)`. Is there a better way?

Anyway, I get the same result with v2.13 b171 that I downloaded from [here](https://github.com/stipub/stixfonts/archive/refs/tags/v2.13b171.zip) (`fontversion: 139592(i)(s)`).

**Comment by firai** [comment]
Seems like this is yet another difference between Word and LaTeX rendering. Pipe brackets (both single and double) are actually a touch too short in Word. I can also confirm that I'm getting similar results as @mdeff with XeLaTeX/LuaLaTeX (via Overleaf) and v2.13 b171.

Word 16 (a.k.a Office 365):
![image](https://user-images.githubusercontent.com/23555/147553404-310f3542-a400-4f87-8e2a-cb7e93bdff1a.png) ![image](https://user-images.githubusercontent.com/23555/147553383-8816a4ea-0778-4e1f-961c-17b31efa0e96.png)

**Comment by tiroj** [comment]
The height of delimiter variant to use is determined by the math layout engine, not directly by the font. It is derived from the overall height of the relevant stacked parts of the equation, but how that is interpreted by the layout engine and a variant selection made is often a bit of a black box from my perspective. And, as noted, not all math layout engines will produce the same results from the same input using the same font.

But this sort of situation where different delimiters are getting different heights around the same content is the sort of thing that suggests a font issue:
![image](https://user-images.githubusercontent.com/5614500/147584624-64c34fbf-cee4-4a50-8144-a3c814f00fe8.png)

**Comment by mdeff** [comment]
I've tried a bunch of delimiters:
![20211228_175618](https://user-images.githubusercontent.com/6806065/147589201-697cd2c5-ccec-4ca4-a901-419f56766d14.png)

Only ‖ is badly scaled.

⧼, ⧽, ⦑, and ⦒ are not scaled at all. (I've never seen them in mathematical texts, but they're mentioned on [Wikipedia](https://en.wikipedia.org/wiki/Bracket_(mathematics)#Symbols_for_representing_angle_brackets).)

**Comment by tiroj** [comment]
In the v2.20, all delimiters now a) have the same set of 12 growing variants, b) are vertically centered on the max axis height, and c) are the same height as each other across the variant set. So, all else being equal, this issue should be fixed in b2.20.

**Comment by mdeff** [comment]
Thanks for the quick fix! Looking forward to v2.20. 🙂
